### ISSUE TITLE
Feature request: ability to scroll through hours by dragging (Dexcom style) and see what bg was at each hour

### ISSUE THREAD

**Issue by Not-AriStienfeld** [nightscout/nightguard#219]
currently the only way to do this is to open the nightscout site, and that site is annoying to navigate on mobile. I would like the ability to easily scroll through previous blood sugar values in a similar fashion to the dexcom and sugarmate apps.

**Comment by dhermanns** [comment]
You mean by long pressing on the nightguard chart?

**Comment by Not-AriStienfeld** [comment]
Long pressing doesn't do anything on my phone

**Comment by dhermanns** [comment]
Yes - not working right now. I don't know the dexcom behavior. So that was just meant to clarify the wished feature.

**Comment by Not-AriStienfeld** [comment]
Yes. That would be awesome
